jormungandr/src/start_up/mod.rs

```diff
@@ -91,33 +91,40 @@ pub fn load_blockchain(
     logger: &Logger,
 ) -> Result<(Blockchain, Tip), Error> {
     use tokio::prelude::*;
+    use tokio_compat::runtime;
+
+    let mut rt = runtime::Builder::new()
+        .name_prefix("load-blockchain-worker-")
+        .core_threads(1)
+        .build()
+        .unwrap();
```

Comment:
you can drop this commit (c60c618e6c1bd2b8016fea4d8a6559b7a4efcd55) in favor of:
https://github.com/input-output-hk/jormungandr/pull/1745